docs/reference/cluster-config.txt

```diff
@@ -480,6 +480,17 @@ The master supports the following configuration settings:
          ``p3.8xlarge``, ``p3.16xlarge``, or ``p3dn.24xlarge``. Defaults
          to ``p3.8xlarge``.
 
+      -  ``spot``: Whether to use spot instances. Defaults to ``false``.
+
+      -  ``spot_max_price``: Optional field indicating the maximum price
+         per hour that you are willing to pay for a spot instance. The
+         market price for a spot instance varies based on supply and
+         demand. If the market price exceeds the ``spot_max_price``,
+         Determined will not launch instances. This field must be a
+         string and must not include a currency sign. For example, $2.50
```

Comment:
question: if a cluster is not in the US, does AWS use the local currency or dollars?

Reply:
I'm not sure. I think it's dollars.